### ISSUE TITLE
R2

### ISSUE THREAD

**Issue by gcblqqqqqqqqqqqqqqq** [Mohamedelrefaie/DrivAerNet#12]
Hello,The model I trained is RedgCNN, and the dataset used is DrivAerNet_STLs_Combined. The R2 value has consistently been negative and LOSS is decreasing and the performance is decent.

**Comment by Mohamedelrefaie** [comment]
Could you please share the learning curves (training and val loss over the number of epochs)?
We have recently re-trained the model and were able to reproduce the results reported in both the DrivAerNet and DrivAerNet++ papers. For reference, please refer to the main paper for the training hyperparameters used.

**Comment by cblgg** [comment]
![图片1](https://github.com/user-attachments/assets/f516e332-2c59-4b02-b719-1695a08d0673)

And  “ Epoch 1 Training Loss: 0.079081 Time: 4061.08s
Average Training R²: -368.4631”
“Epoch 1 Validation Loss: 0.0161, Avg Inference Time: 0.3453s
Average Validation R²: -58.6274”parameters  are:      'exp_name': 'CdPrediction_DrivAerNet_r2_100epochs_5k',
    'cuda': True,
    'seed': 1,
    'num_points': 5000,
    'lr': 0.001,
    'batch_size': 4,
    'epochs': 100,
    'dropout': 0.4,
    'emb_dims': 512,
    'k': 40,
    'optimizer': 'adam',
